src/smc_sagews/smc_sagews/graphics.py

```diff
@@ -314,13 +314,16 @@ def parse_mtl(p):
     # Conversion functions
     #####################################
 
-    def convert_index_face_set_test(p, T, extra_kwds):
+    def convert_index_face_set(p, T, extra_kwds):
+        p.obj()  # just to ensure one call to .triangulate if needed
         if T is not None:
-            p = p.transform(T=T)
-        p.triangulate()
-        face_geometry = [{"material_name": p.texture.id, "faces": [[int(v) for v in f] for f in p.index_faces()]}]
-        vertex_geometry = [json_float(t) for v in p.vertices() for t in v]
+            vertices = [T.transform_point(v) for v in p.vertices()]
+        else:
+            vertices = p.vertices()
+        face_geometry = [{"material_name": p.texture.id,
+                          "faces": [[int(v) for v in f] for f in p.index_faces()]}]
```

Comment:
Replacing ```"faces": [[int(v) for v in f] for f in p.index_faces()]}]``` by
```"faces": [[int(v)+1 for v in f] for f in p.index_faces()]}]``` makes drawing normal 3d plots not break.

HOWEVER, what is the point of this patch?  What is it doing?  Why is it even here?  What's an example to illustrate this working or not working?  I can't even find anything in my email about this, etc.